internal/handler/auth/auth.go

```diff
@@ -387,10 +390,25 @@ func SendPasswordResetLink(c *gin.Context) {
 	// add user_id in password reset link
 	resetLink := fmt.Sprintf("%s?user_id=%s", viper.GetString("auth.password_reset_link"), user.User.UserID.String())
 
-	log.Print("Reset_Password_link:", resetLink)
-
 	// TO-DO send link on the registered/verified email
 
+	// add the password reset link and also its expiry time.
+	userVerificationRepo := env.FromContext(c).UserVerificationRepository
+	updateUserVerification := &userverification.UpdateUserRequest{
+		UserID: user.User.UserID,
+		Values: map[string]interface{}{
+			"reset_password_url":            resetLink,
```

Comment:
we should not store whole link in column, instead - there should be a unique value (can be large nanoid) that should be stored and appended in reset link (instead of user id)

If the only variable in reset password link is user id then I can guess reset-pass-link of any other user as well and we're only checking link expiry time as validation.

Also, can please add few basic checks on password, like - min 8 characters, alphanumeric, 1 special character (1 common at both times - create and reset)